test/BuildingRegistry.Tests/BackOffice/Api/WhenAttachingAddressToBuildingUnit/GivenAddressRemoved.cs

```diff
@@ -0,0 +1,59 @@
+namespace BuildingRegistry.Tests.BackOffice.Api.WhenAttachingAddressToBuildingUnit
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AutoFixture;
+    using Building;
+    using Building.Exceptions;
+    using BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Requests;
+    using BuildingRegistry.Api.BackOffice.BuildingUnit;
+    using FluentAssertions;
+    using FluentValidation;
+    using Moq;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class GivenAddressRemoved : BackOfficeApiTest
```

Comment:
This test class does the same as GivedAddress**Is**Removed, no?

Reply:
true